backend/controllers/taskController.js

```diff
@@ -15,4 +19,101 @@ const createTask = asyncHandler(async (req, res) => {
   }
 });
 
-module.exports = { createTask };
+const getTasks = asyncHandler(async (req, res) => {
```

Comment:
Since this is a `GET` request, we shouldn't be extracting the `user_id` from the body. This should already be passed in from the JWT as we mentioned last week

Reply:
Fixed: Changes are reflected in recent commits